libraries/AP_HAL_ChibiOS/hwdef/TMotorH743/README.md

```diff
@@ -0,0 +1,94 @@
+# T-Motor H7 Mini Flight Controller
+
+The T-Motor H7 Mini is a flight controller produced by [T-Motor](https://store.tmotor.com/goods-1295-H7+MINI.html).
+
+## Features
+
+ - MCU - STM32H743 32-bit processor running at 480 MHz
+ - IMU - ICM42688/BMI270
+ - Barometer - DPS310
+ - OSD - AT7456E
+ - Onboard Flash: 128Mbits
+ - 6x UARTs (1,5,6,7,9)
+ - 9x PWM Outputs (8 Motor Output, 1 LED)
+ - Battery input voltage: 2S-6S
+ - BEC 5V/2A, 10V/1.5A
+
+## Pinout
+
+![T-Motor H7 Mini Board](TMotorH7Mini_Board.JPG "T-Motor H7 Mini")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (ESC Telemetry)
+ - SERIAL3 -> UART3
```

Comment:
where is this pinned out?

Reply:
two pads in the middle of the board (see the diagram)

Reply:
totally missed those,sorry